templates/overrides/semantic/webbasic/wwwroot/css/site.css

```diff
@@ -0,0 +1,116 @@
+.hidden.menu {
+    display: none;
+}
+
+.masthead {
+    margin-top: 15px !important;
+    margin-bottom: 15px !important;
+}
+
+    .masthead.segment {
+        min-height: 700px;
+        padding: 1em 0em;
+    }
+
+    .masthead .logo.item img {
+        margin-right: 1em;
+    }
+
+    .masthead .ui.menu .ui.button {
+        margin-left: 0.5em;
+    }
+
+    .masthead h1.ui.header {
+        margin-top: 3em;
+        margin-bottom: 0em;
+        font-size: 4em;
+        font-weight: normal;
+    }
+
+    .masthead h2 {
+        font-size: 1.7em;
+        font-weight: normal;
+    }
+
+.ui.vertical.stripe {
+    padding: 8em 0em;
+}
+
+    .ui.vertical.stripe h3 {
+        font-size: 2em;
+    }
+
+    .ui.vertical.stripe .button + h3,
+    .ui.vertical.stripe p + h3 {
+        margin-top: 3em;
+    }
+
+    .ui.vertical.stripe .floated.image {
+        clear: both;
+    }
+
+    .ui.vertical.stripe p {
+        font-size: 1.33em;
+    }
+
+    .ui.vertical.stripe .horizontal.divider {
+        margin: 3em 0em;
+    }
+
+.quote.stripe.segment {
+    padding: 0em;
+}
+
+    .quote.stripe.segment .grid .column {
+        padding-top: 5em;
+        padding-bottom: 5em;
+    }
+
+.footer.segment {
+    padding: 5em 0em;
+    margin-top: 15px !important;
+}
+
+.secondary.pointing.menu .toc.item {
+    display: none;
+}
+
+.ui.top.menu .toc.item {
+    display: none;
+}
+
+
+@media only screen and (max-width: 700px) {
+    .ui.top.menu .toc.item {
+        display: block;
+    }
+
+    .ui.top.menu .item {
+        display: none;
+    }
+}
+
+.login-container {
+    position: relative !important;
+}
+
+.content {
+    padding-top: 15px;
+}
+
+.pusher {
+    padding-top: 53px;
+}
+
+.ui.sidebar.menu {
+    padding-top: 53px;
+}
+
+/*Hack: http://stackoverflow.com/questions/19810966/semanitc-ui-with-asp-net-mvc-checkbox*/
```

Comment:
This hides problem - people should be aware of different solution(s):
https://github.com/aspnet/Mvc/issues/2369